src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -30,7 +30,10 @@
 
 /**
  * <p>
- * Checks the indentation of the continuation lines in at-clauses.
+ * Checks the indentation of the continuation lines in at-clauses. That is whether the continued
+ * description of at clauses should be intended or not. If the text is not properly intended it
```

Comment:
`intended` => `indented`
Happens twice on this line.

Reply:
Done